lib/modules/manager/terraform/extractors/others/modules.spec.ts

```diff
@@ -69,7 +69,7 @@ describe('modules/manager/terraform/extractors/others/modules', () => {
         'ssh://github.com/hashicorp/example?ref=v1.0.0&depth=1',
       )?.groups;
       const folder = gitTagsRefMatchRegex.exec(
-        'git::ssh://git@git.example.com/modules/foo-module.git//bar?depth=1&ref=v1.0.0',
+        'git::ssh://git@git.example.com:modules/foo-module//bar?depth=1&ref=v1.0.0',
```

Comment:
Was it necessary to change this existing test? We normally prefer adding tests or adding to tests, instead of changing existing ones (which can hide regressions)